### DISCUSSION TITLE
New feature

### DISCUSSION THREAD

**Discussion by Myselforsomethingsimilar** [Brionjv/Tomodachi-Life-Save-Editor#109] [Ideas]
Hey. I used this Save Editor quite a lot of times, and I wanted to suggest you a feature you might want to add...
I don't actually know if there is this feature, but I'd like some input/button to mark the Mii asleep/awake. I was playing at night and it kinda bothered me everyone was asleep, so yeah... I hope you like my idea.

**Comment by Brionjv** [comment]
Hi,I'll try to do this
